etc/profile-a-l/ani-cli.profile

```diff
@@ -0,0 +1,41 @@
+# Firejail profile for ani-cli
+# Description: Shell script to watch Anime from the terminal
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include ani-cli.local
+# Persistent global definitions
```

Comment:
Now that this is a redirect profile, it's important to `avoid including globals.local twice`. Here's what we use in other profiles for that purpose:
```
# added by included profile
#include globals.local
```